common/PersistenceFormat.js

```diff
@@ -18,17 +18,17 @@ module.exports = {
         return entryId;
     },
 
-    getFlavorFullPath: function (entryId, bitrate) {
+    getFlavorFullPath: function (entryId, flavorRecognizer) {
         var config = require('./Configuration');
-        return path.join(config.get('rootFolderPath'), entryId, bitrate.toString());
+        return path.join(config.get('rootFolderPath'), entryId, flavorRecognizer.toString());
```

Comment:
performance issue!!

var config = require('./Configuration');
 return path.join(config.get('rootFolderPath')

can be done once on the beginning of ifle